packages/big-design/src/components/StatefulTable/StatefulTable.tsx

```diff
@@ -62,6 +85,8 @@ const InternalStatefulTable = <T extends TableItem>({
   itemName,
   items = [],
   keyField,
+  localization: localizationProp,
```

Comment:
might want to keep the style across the change:

```suggestion
  localization = defaultLocalization,
```